packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx

```diff
@@ -216,4 +217,58 @@ describe('HCP home page', () => {
 
     expect(invitePatientButton).toBeEnabled()
   })
+
+  it('should check that the team creation button is valid if all fields are complete', async () => {
+    await act(async () => {
+      render(getHomePage())
+    })
+    const teamMenu = screen.getByLabelText('Open team menu')
+    userEvent.click(teamMenu)
+    userEvent.click(screen.getByText('New care team'))
+    const dialogTeam = screen.getByRole('dialog')
+    const createTeamButton = within(dialogTeam).getByRole('button', { name: 'Create team' })
+    const nameInput = within(dialogTeam).getByRole('textbox', { name: 'Name' })
+    const adress1Input = within(dialogTeam).getByRole('textbox', { name: 'Address 1' })
+    const adress2Input = within(dialogTeam).getByRole('textbox', { name: 'Address 2' })
+    const zipcodeInput = within(dialogTeam).getByRole('textbox', { name: 'Zipcode' })
+    const cityInput = within(dialogTeam).getByRole('textbox', { name: 'City (State / Province)' })
+    const phoneNumberInput = within(dialogTeam).getByRole('textbox', { name: 'Phone number' })
+    const prefixPhoneNumber = within(dialogTeam).getByText('+33')
+    const emailInput = within(dialogTeam).getByRole('textbox', { name: 'Email' })
+
+    // Team creation button disabled and all fields empty
+    expect(nameInput).toHaveTextContent('')
+    expect(adress1Input).toHaveTextContent('')
+    expect(adress2Input).toHaveTextContent('')
+    expect(zipcodeInput).toHaveTextContent('')
+    expect(cityInput).toHaveTextContent('')
+    expect(phoneNumberInput).toHaveTextContent('')
+    expect(emailInput).toHaveTextContent('')
+    expect(dialogTeam).toBeInTheDocument()
+    expect(prefixPhoneNumber).toHaveTextContent(PhonePrefixCode.FR)
+    expect(createTeamButton).toBeDisabled()
+
+    // Team dropdown with prefix phone number
+    fireEvent.mouseDown(within(screen.getByTestId('team-edit-dialog-select-country')).getByRole('button'))
+    fireEvent.click(screen.getByRole('option', { name: 'Austria' }))
+    expect(prefixPhoneNumber).toHaveTextContent(PhonePrefixCode.AT)
+
+    // Team creation button activated and all fields filled
+    await userEvent.type(nameInput, lastName)
+    await userEvent.type(adress1Input, '5 rue')
+    await userEvent.type(zipcodeInput, '38000')
+    await userEvent.type(cityInput, 'Grenoble')
+    await userEvent.type(phoneNumberInput, '0600000000')
+    await userEvent.type(emailInput, 'toto@titi.com')
+    expect(createTeamButton).not.toBeDisabled()
+
+    // Team creation button disabled and a field in error with the error message
+    userEvent.clear(phoneNumberInput)
+    userEvent.clear(emailInput)
+    await userEvent.type(phoneNumberInput, '060')
+    expect(within(dialogTeam).getByText('Please enter a valid phone number')).toBeVisible()
+    await userEvent.type(emailInput, 'tototiti.com')
+    expect(within(dialogTeam).getByText('Invalid email address (special characters are not allowed).')).toBeVisible()
+    expect(createTeamButton).toBeDisabled()
```

Comment:
I would then fill the phone number, the email and the zip code properly, check that the button is enabled, click on the create button and check that the API call has been made :slightly_smiling_face:

Reply:
Okay but this part is made to test the cases where the inputs of the form are incorrect and with the disable button

Reply:
I then add above the test with the API call

Reply:
If it is easily possible, it is better to do everything in one test. The more tests we have, the longer it takes (as you have to initialize the DOM in each test)